src/validation/validationLottoPrice.test.js

```diff
@@ -0,0 +1,38 @@
+import validationLottoPrice from "./validationLottoPrice.js";
+import CustomError from "../CustomError.js";
+import { ERROR_MESSAGE } from "../constants/message.js";
+
+describe("validationLottoPrice 유효성 검사", () => {
+  test("1000.1원은 정수가 아니다.", () => {
+    const price = 1000.1;
+    expect(() => validationLottoPrice(price)).toThrow(
+      new CustomError(ERROR_MESSAGE.INVALID_INTEGER)
+    );
+  });
+
+  test("5001원은 1000원 단위로 나눠지지 않는다.", () => {
+    const price = 5001;
+    expect(() => validationLottoPrice(price)).toThrow(
+      new CustomError(ERROR_MESSAGE.INVALID_MULTIPLE_OF_THOUSAND)
+    );
+  });
+
+  test("999원은 최소 구입 금액(1000원)보다 작다.", () => {
+    const price = 999;
+    expect(() => validationLottoPrice(price)).toThrow(
+      new CustomError(ERROR_MESSAGE.INVALID_MIN_PRICE)
+    );
+  });
+
+  test("101,000원은 최대 구입 금액(10만원)을 초과한다.", () => {
```

Comment:
100,000원 처럼 쭉 통일 해주세요!

Reply:
```suggestion
  test("101,000원은 최대 구입 금액(100,000원)을 초과한다.", () => {
```
테스트 이름에 10만원을 100,000원으로 표현하는걸 말씀하시는게 맞을까요??

Reply:
네 맞습니다!